lib/experimental.ts

```diff
@@ -18,8 +18,9 @@ export async function experimentalAnalysis(
   const imageType = getImageType(targetImage);
   switch (imageType) {
     case ImageType.DockerArchive:
-      return dockerArchive(targetImage, dockerfileAnalysis);
-
+      return localArchive(targetImage, imageType, dockerfileAnalysis);
+    case ImageType.OciArchive:
+      return localArchive(targetImage, imageType, undefined);
```

Comment:
Should we pass in here the dockerfileAnalysis too?

Reply:
Omg, I think I got this wrong? Can we have a dockerfile analysis here?

Reply:
The analysis was already done earlier (first step of calling `inspect()`), just not sure what it's used for but I think it should be included in the function call!